test/manifolds/torus_example.jl

```diff
@@ -0,0 +1,114 @@
+using Revise
```

Comment:
this could be reworked into a Pluto notebook, I can help with that (and include it into the docs in the tutorials PR later).

Reply:
Yes, it would be great if you could turn it into a Pluto notebook :slightly_smiling_face: .

Reply:
Sure. My idea would be – finish this PR with the example here as is and I will rework it in the #534 afterwards?

Reply:
Sure, that's a good plan.